system/lib/libc/musl/arch/emscripten/bits/stat.h

```diff
@@ -1,22 +1,20 @@
-/* copied from kernel definition, but with padding replaced
- * by the corresponding correctly-sized userspace types. */
-
 struct stat
```

Comment:
what `struct stat` definition is this mirroring? is it the stat used in x32?

Reply:
Yes, it's the x32 linux ABI. I've now added a comment.